website/source/docs/jobops/index.html.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: "docs"
+page_title: "Operating a Job"
+sidebar_current: "docs-jobops"
+description: |-
+  Learn how to operate a Nomad Job.
+---
+
+# Operating a Job
+
+Once a job has been submitted to Nomad, users must be able to inspect the state
```

Comment:
Its more of saying when operating a service you must have access to these things